echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/WebhookEventMonitor.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2016 Google, Inc.
```

Comment:
`Google, Inc.` probably isn't correct to use in the copyright. 

Thinking `Copyright 2016 The original authors` but just checking what we need.